app/src/main/res/drawable/ic_private_browsing.xml

```diff
@@ -9,5 +9,5 @@
     android:viewportHeight="24">
     <path
         android:fillColor="@color/photonWhite"
```

Comment:
This is also equivalent to mozac_ui_icons_fill. Makes you wonder if we should do something to replace all of these android:fillColor="@color/photonWhite" with mozac_ui_icons_fill. Maybe file an issue?

Reply:
We could address this in another ticket, as it's not related to this PR, and there we can discuss if we want to change that globally.

Reply:
I filed https://github.com/mozilla-mobile/android-components/issues/10164 for continuing the discussion